com.unity.ml-agents/Runtime/Startup.cs

```diff
@@ -4,7 +4,7 @@
 
 namespace MLAgents
 {
-    public class Startup : MonoBehaviour
+    internal class Startup : MonoBehaviour
```

Comment:
Do we use this anywhere? It feels like it should be in the examples code.

Reply:
ah yeah.  That's my bad.  I can move it after this PR lands.

Reply:
@xiaomaogy

Reply:
We use this script for switching the scenes before the training starts. So not really belong to the examples code. Should live where the Startup scene lives.

Reply:
But the startup scene is in the example environments

Reply:
yeah, above I said I should have moved it to the `Project` folder.